content/events/2019-02-14-candygrams.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "CSSS: Valentine's Candygrams"
+date: 2019-01-30
+start_date: 2019-02-14T00:00:00.000Z
+end_date: 2019-02-14T23:59:00.000Z
+categories: [Event]
+tags: [Candygrams]
+author: Hai Lin Zhang
+location:
+  name: ICICS 021
+  address:
+    streetAddress: 2366 Main Mall
+    addressLocality: Vancouver
+    postalCode: V6T 1Z4
+    addressRegion: BC
+    addressCountry: CA
+images:
+  - /files/candygrams_2019.jpg
```

Comment:
Let's use the SVG banner instead. SVG is preferable because it looks better and uses less bytes.